inc/wp-forms-api.js

```diff
@@ -161,8 +161,14 @@
 				view.model.fetch();
 
 				// Don't save input name as part of the model as it should be invariant
-				view.input_name = $(this).find('input').attr('name');
-				view.input_type = $(this).data('attachment-type');
+				if(typeof this.name === 'undefined'){
+					// New widgets added to sidebar after page load
```

Comment:
Cool - had the single property syntax on the first try, which also had the old `data.input_name = this.name` - I attributed the blank values to the wrong code at first